ai21/clients/common/assistant/thread.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import annotations
+
+from abc import ABC, abstractmethod
+from typing import List
+
+from ai21.models.responses.thread_response import ThreadResponse, CreateMessagePayload
+
+
+class Thread(ABC):
+    _module_name = "threads"
+
+    @abstractmethod
+    def create(
+        self,
+        messages: List[CreateMessagePayload],
+        **kwargs,
+    ) -> ThreadResponse:
+        pass
+
```

Comment:
Do we also want to support something like this? https://github.com/openai/openai-python/blob/main/src/openai/resources/beta/threads/messages.py#L32

where we also have `messages` method inside `Threads` class?

Reply:
yeah, that's the next PR :)